### DISCUSSION TITLE
openlitespeed + wp + lscache

### DISCUSSION THREAD

**Discussion by slypire** [coollabsio/coolify#3307] [Service Template Requests]
https://docs.litespeedtech.com/cloud/docker/ols-wordpress/
https://github.com/litespeedtech/openlitespeed

Would be super nice to have this along nginx option, for us who use WP and host client websites.
Litespeed is currently the fastest when it comes to wordpress.
when setting it up, there should also be an option to choose php version.

Update: 
Here is a one click litespeed script that might be useful: https://github.com/litespeedtech/ols1clk

**Comment by ptrsmk** [comment]
This seems like a no-brainer. This would be fantastic.

**Comment by mredodos** [comment]
+1

**Comment by mredodos** [comment]
check this is literally ready with compose 

https://docs.litespeedtech.com/cloud/docker/ols-wordpress/
https://github.com/litespeedtech/ols-docker-env/blob/master/docker-compose.yml

i think is better remove acme probably coolify have traefik caddy for certificate

**Comment by slypire** [comment]
Hey @peaklabs-dev , is there any chance you could take a look at this 😳

**Comment by mredodos** [comment]
Meanwhile, I'm trying to build an image with all wordpress openlightspeed, redis, relay for maximum performance

**Reply by Ziltosh** [reply]
Hello, have you been able to make progress on this? I am looking for exactly the same thing.

**Reply by mredodos** [reply]
im working on, im try to add some security features but handle multiple issue https://hub.docker.com/repository/docker/anideaforbusiness/wordpress-ols-super-optimized-eg/general

actualy not work but im testing on local for fix the issue.. 

Unfortunately, I'm also busy with jobs and I can't devote myself 100% so it's taking me a long time

**Reply by wpdevninja** [reply]
Is there any update @mredodos?

**Comment by mrtornado** [comment]
I'm looking for this also .. any progress, I keep getting stuck at: 
```
This page isn’t working
.... redirected you too many times.

Try deleting your cookies.
ERR_TOO_MANY_REDIRECTS
```

**Comment by mredodos** [comment]
https://hub.docker.com/repository/docker/anideaforbusiness/wordpress-ols-super-optimized-eg/general
https://github.com/mredodos/wordpress-superoptimized-v2

actually i have a lot of issue with fail2ban the purpose is create a very good image with best performance and best security...

But as I analyzed and worked on the image, I realized that it is probably not the best solution

so personally on work I decided to get a server and install aapanel which allows much more specific and granular management to manage and host wordpress with a whole host of excellent security features by default. 

as i manage (work permitting) i delete file2ban from the image so it should work is all public if you want change stuf and so you can

**Comment by nityam2007** [comment]
Guys any updates ??

**Comment by slypire** [comment]
Show us, wordpress users, some love @peaklabs-dev @andrasbacsai

**Comment by dmvdev01** [comment]
Any updates, I need this as well! OpenLiteSpeed is the best for WP!!

**Comment by rodsotdia** [comment]
This would be awesome! I miss to have Litespeed implemented

**Comment by idevN** [comment]
Please, can we have this on Cooligy?

**Comment by EmirAzaiez** [comment]
If you're still searching how to put OLS with coolify, I found this blogpost : https://hasto.pl/wordpress-on-coolify-100-pagespeed-with-litespeed/

I’m still struggling to get LSCache working properly. I’ll let you know if I find anything interesting!
